docs/source/tutorials/GEDI/GEDI_L3_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "show_thumbinal(download_path)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5c9c6e5e",
+   "metadata": {},
+   "source": [
+    "**[OPTIONAL] Overlay raster layer on top of folium map**\n",
+    "\n",
+    "To make proper visualization of canopy heights, we need to visualize the tiff image in a map. Since the tif file is heavy, it is too compute intensive to load and may cause kernel to die due to insufficient memory and compute. So this part is optional. "
```

Comment:
```suggestion
    "To properly visualize the canopy heights, we need to display the TIF image on the map. The TIF image file may be too memory and compute-intensive for the kernel causing the process to exit."
```
We can get rid of the optional portion if it's in the section heading

Reply:
Sounds good